src/ProxyManager/Configuration.php

```diff
@@ -27,14 +27,14 @@ class Configuration
 {
     public const DEFAULT_PROXY_NAMESPACE = 'ProxyManagerGeneratedProxy';
 
-    protected ?string $proxiesTargetDir;
-    protected string $proxiesNamespace = self::DEFAULT_PROXY_NAMESPACE;
-    protected ?GeneratorStrategyInterface $generatorStrategy;
-    protected ?AutoloaderInterface $proxyAutoloader;
-    protected ?ClassNameInflectorInterface $classNameInflector;
-    protected ?SignatureGeneratorInterface $signatureGenerator;
-    protected ?SignatureCheckerInterface $signatureChecker;
-    protected ?ClassSignatureGeneratorInterface $classSignatureGenerator;
+    protected ?string $proxiesTargetDir                                  = null;
```

Comment:
These did not have a default value by design 🤔

Reply:
(to be checked - it may have been wrong upfront)

Reply:
Looking at this: why the heck did I make all this stuff `protected`? :| 

Default value is good though: let's roll with it.